frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/general-tab/simulated-region-overview-general-tab.component.html

```diff
@@ -1,13 +1,157 @@
+                (click)="vehiclesCollapsed = !vehiclesCollapsed"
+                style="cursor: pointer"
+            >
+                <th scope="rowgroup" class="w-75">
+                    <i
+                        *ngIf="vehiclesCollapsed; else vehiclesExpanded"
+                        class="bi bi-caret-right-fill"
+                    ></i>
+                    <ng-template #vehiclesExpanded
+                        ><i class="bi bi-caret-down-fill"></i
+                    ></ng-template>
+                    Fahrzeuge
+                </th>
+                <th class="w-25">
+                    {{ ((vehicles$ | async) ?? {})['all']?.length }}
+                </th>
+            </tr>
+            <tr
+                *ngFor="let template of vehicleTemplates$ | async"
+                #collapse="ngbCollapse"
+                [(ngbCollapse)]="vehiclesCollapsed"
+            >
+                <td class="ps-4">{{ template.vehicleType }}</td>
+                <td>
+                    {{
+                        ((vehicles$ | async) ?? {})[template.vehicleType]
+                            ?.length
+                    }}
+                </td>
+            </tr>
+        </tbody>
+        <tbody>
+            <tr
+                (click)="personnelCollapsed = !personnelCollapsed"
+                style="cursor: pointer"
+            >
+                <th scope="rowgroup" class="w-75">
+                    <i
+                        *ngIf="personnelCollapsed; else personnelExpanded"
+                        class="bi bi-caret-right-fill"
+                    ></i>
+                    <ng-template #personnelExpanded
+                        ><i class="bi bi-caret-down-fill"></i
+                    ></ng-template>
+                    Personal
+                </th>
+                <th class="w-25">{{ (personnel.all$ | async)?.length }}</th>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="personnelCollapsed">
+                <td class="ps-4">Notarzt</td>
+                <td>{{ (personnel.notarzt$ | async)?.length }}</td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="personnelCollapsed">
+                <td class="ps-4">Notfallsanitäter</td>
+                <td>{{ (personnel.notSan$ | async)?.length }}</td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="personnelCollapsed">
+                <td class="ps-4">Rettungssanitäter</td>
+                <td>{{ (personnel.rettSan$ | async)?.length }}</td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="personnelCollapsed">
+                <td class="ps-4">Sanitäter</td>
+                <td>{{ (personnel.san$ | async)?.length }}</td>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="personnelCollapsed">
+                <td class="ps-4">Gruppenführer</td>
+                <td>{{ (personnel.gf$ | async)?.length }}</td>
+            </tr>
```

Comment:
```suggestion
            <tr *ngFor="let categoryKey of personnelCategoriesMap | keys" #collapse="ngbCollapse" [(ngbCollapse)]="personnelCollapsed">
                <td class="ps-4">{{personnelCategoriesMap[categoryKey]}}</td>
                <td>{{ (personnel[categoryKey] | async)?.length }}</td>
            </tr>
```

and include this in the components class:
```ts
    public readonly personnelCategoriesMap = {
        gf$: 'Gruppenführer',
        notarzt$: 'Notarzt',
        notSan$: 'Notfallsanitäter',
        rettSan$: 'Rettungssanitäter',
        san$: 'Sanitäter',
    };
```

Reply:
Done (with a slightly different implementation) in https://github.com/hpi-sam/digital-fuesim-manv/pull/721/commits/76e09b1dd6cfab60a5d26053467116bc7aa4e989